Valet/VALValet.m

```diff
@@ -450,6 +450,10 @@ - (BOOL)setObject:(nonnull NSData *)value forKey:(nonnull NSString *)key options
             keychainData[(__bridge id)kSecValueData] = value;
             
             status = VALAtomicSecItemAdd((__bridge CFDictionaryRef)keychainData, NULL);
+            
+            if (status == errSecMissingEntitlement) {
+                NSLog(@"A 'Missing Entitlements' error occurred. This is likely due to an Apple Keychain bug. As a workaround try running on a device that is not attached to a debugger.\n\nMore information: https://forums.developer.apple.com/thread/4743\n");
```

Comment:
It just looked better in the console. Otherwise the last line with the forums link gets lost in the next message.